wiki/Modding/en.md

```diff
@@ -1,262 +1,42 @@
 
-If you spot anything weird while the storyboard is playing (e.g. bad rendering of pictures, emptiness, etc.), it is a good idea to tell the [storyboarder](/wiki/storyboarder) so that they can fix it.
-You can check out the storyboard by pressing `F2` or clicking the `Design` tab at the top of the screen in editor.
+Each queue's creator dictates the queue's rules, controlling which beatmaps are viable for receiving mods. Queues are usually open for requests in two ways:
 
-The dimensions of the elements should not exceed the default size.
+- **Normal mod (NM):** A mod that will be provided without work from the requesting beatmapper.
+- **Mod for mod (M4M):** A trade that requires each user to mod the other's beatmap.
 
-### Preview Point
+The `#modreqs` chat channel is populated by beatmappers posting maps they want modded. Most beatmaps posted do not receive mods.
 
-All maps must have a preview point.
+Well-known modders often note whether or not they are accepting mod requests on their user profiles.
 
-A preview point will be denoted by a yellow tick mark on the [timeline](/wiki/timeline).
-You can also check if a map has a preview point by going into [AiMod](/wiki/AiMod) and checking under the `Style` tab.
+Searching the pending beatmaps listing for beatmaps to mod is not recommended because most submissions are not aiming for Ranked status, so their creators will potentially not respond to mods.
 
-The preview point must be the same in all difficulties.
+## Modding process
 
-### Folder Organization
+To mod a beatmapset, start by downloading the most up-to-date version and opening its discussion page. Play the difficulty you want to mod and keep in mind any gameplay-related issues. When completed, open the difficulty in the editor and analyze it for potential issues.
 
-To see the beatmapset folder, click `File` then `Open Song Folder`.
+When you come across an issue, select any relevant hit objects, then use the copy shortcut (`Ctrl` + `C`) to copy their timestamp information as text. On the beatmap's discussion page, locate the difficulty being modded, navigate to the applicable tab, paste (`Ctrl` + `V`) any applicable timestamps and your feedback. The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties on a beatmapset.
 
-The song's bit rate must be between 128KBps and 192 KBps.
+All mod posts should address what the issue is, why it is an issue, and how to fix it. For example, if a beatmap has an unnecessarily large jump, a mod could potentially be worded as "The spacing between these objects is too high. The song is not intense enough to justify this size of a jump, so moving the objects closer together would be more appropriate!". Many mod posts include links to images so that beatmappers can better understand any suggestions.
```

Comment:
i feel like there's somewhere else to link for this (is it modding coc? i think? )